README.md

```diff
@@ -111,22 +111,22 @@ kubectl apply -f sample_assets/secretBank.yaml
 
 ## Define policies
 
-You can define OpenPolicyAgent policy to apply them to datasets. You can follow the `Define data access policies` section in [this example](https://607573df9860bf9afcf4805b--mesh-for-data.netlify.app/samples/notebook/#define-data-access-policies).
+You can define OpenPolicyAgent policy to apply them to datasets. You can follow the `Define data access policies` section in [this example](https://fybrik.io/dev/samples/notebook/).
 
 ## Deploy M4D application which triggers module
-1. In `m4dapplication.yaml`:
+1. In `fybrikapplication.yaml`:
     - Change `metadata.name` to your application name.
     - Define `appInfo.purpose`, `appInfo.role`, and `spec.data`.
     - Change `data.dataSetID` field to the identifier of the asset in the catalog which is in the format `<namespace>/<name>`.
  
-2.  Deploy `M4DApplication` in `default` namespace:
+2.  Deploy `FybrikApplication` in `default` namespace:
 ```bash
-kubectl apply -f m4dapplication.yaml -n default
+kubectl apply -f fybrikapplication.yaml -n default
 ```
 3.  Check if `M4DModule` successfully deployed:
```

Comment:
Should `M4DModule` be changed to `FybrikModule`?

Reply:
Thanks Fixed